includes/widgets/tabs.php

```diff
@@ -74,6 +74,17 @@ public function get_keywords() {
 		return [ 'tabs', 'accordion', 'toggle' ];
 	}
 
+	protected function get_initial_config() {
+		$config = parent::get_initial_config();
+
+		$config['deprecated'] = [
```

Comment:
@matipojo it would be nice if we cloud have document for all widget config options.

Reply:
Extract to a method `mark_as_deprecated( $config )` on the `Contorls_Stack` so it will be a part of the infra.

Reply:
Also, mark it as deprecated only if the new experiment is active